src/test/java/uk/gov/hmcts/reform/professionalapi/repository/OrganisationRepositoryTest.java

```diff
@@ -47,12 +33,21 @@ public void test_findByOrganisationIdentifier() {
 
     @Test
     public void test_findByStatus() {
-        Organisation activeOrg = new Organisation("Org-Name", OrganisationStatus.ACTIVE, "sra-id", "companyN", false, "www.org.com");
-        organisationRepository.save(activeOrg);
-
         List<Organisation> activeOrganisations = organisationRepository.findByStatus(OrganisationStatus.ACTIVE);
 
         assertThat(activeOrganisations).hasSize(1);
-        assertThat(activeOrganisations.get(0)).isEqualTo(activeOrg);
+        assertThat(activeOrganisations.get(0)).isEqualTo(organisation);
+    }
+
+    @Test
+    public void test_findByName() {
+        Organisation org = organisationRepository.findByName(organisation.getName());
+        assertThat(org).isEqualTo(organisation);
+    }
```

Comment:
have you written the custom equalsto() method to compare two objects of organisation?